mozetl/taar/taar_similarity.py

```diff
@@ -298,13 +354,15 @@ def get_lr_curves(spark, features_df, cluster_ids, kernel_bandwidth,
 
 @click.command()
 @click.option('--date', required=True)
-@click.option('--bucket', default='telemetry-parquet')
-@click.option('--prefix', default='taar/similarity/')
-@click.option('--num_clusters', default=20)
-@click.option('--num_donors', default=1000)
-@click.option('--kernel_bandwidth', default=0.35)
-@click.option('--num_pdf_points', default=1000)
-def main(date, bucket, prefix, num_clusters, num_donors, kernel_bandwidth, num_pdf_points):
+@click.option('--bucket', default='telemetry-parquet', show_default=True)
+@click.option('--prefix', default='taar/similarity/', show_default=True)
+@click.option('--num_clusters', default=20, show_default=True)
+@click.option('--num_donors', default=1000, show_default=True)
+@click.option('--kernel_bandwidth', default=0.35, show_default=True)
+@click.option('--num_pdf_points', default=1000, show_default=True)
+@click.option('--whitelist', default=WHITELIST.BASIC, show_default=True)
```

Comment:
@mlopatka the whitelist is specified here as a default option in the click decorator.  There are 3 options for the whitelist all specified here:

https://github.com/mozilla/python_mozetl/pull/240/files#diff-5b9c8e28072d416224a542ad4cbb0256R25